Assets/Scripts/Input/InputActionsReader.cs

```diff
@@ -17,6 +17,7 @@ public class InputActionsReader : MonoBehaviour
     public Vector2GameEvent gameplayLookEvent;
     public VoidGameEvent gameplayInteractPressed;
     public VoidGameEvent gameplayInteractReleased;
+    public BooleanGameEvent gameplayJumpPressed;
```

Comment:
A new `GameEvent` to raise `true`/`false` when the `jump` action is pressed/released! 

The `BooleanGameEvent` itself is `Assets/ScriptableObjects/GameEvents/Input/GameplayJumpEventSO.asset`.